python/manageJobs.py

```diff
@@ -260,7 +280,39 @@ def manageJobs(argv=None):
                 # actions that can be applied to all jobs
                 jobnums = [j.num for j in jobs]
                 schedd.act(htcondor.JobAction.Remove,jobnums)
+            elif options.xrootdResubmit:
+                print ""

```

Comment:
behind quiet?

Reply:
`quiet` was really implemented to avoid the printing of the jobs list (what happens right now). It wasn't meant to modify/silence `xrootdResubmit` printouts.

Reply:
That's not how the option description is written, and I think it's confusing if an option called "quiet" doesn't actually quiet everything.

Reply:
Changed the logic based on the new commit. I removed the `quiet` option. There is still a `verbose` option, but it only applies to the cases involving `-X`.